db/migrate/20231015020025_update_mentee_application_updated_at.rb

```diff
@@ -0,0 +1,12 @@
+class UpdateMenteeApplicationUpdatedAt < ActiveRecord::Migration[7.0]
+  def up
+    UserMenteeApplication.all.each do |application|
+      new_updated_at = application.current_state.updated_at
+      application.update!(updated_at: new_updated_at)
```

Comment:
This is all a bit iffy. N+1 and each update is a separate transaction, but I couldn't immediately ideate a better way to do it while not drifting out of the scope of this ticket. See more in [this thought process doc](https://www.notion.so/toyhammered/UserMenteeApplication-Ordering-b676a58205544cc68c1aa16e268fbac4?pvs=4#95c17b6c01964f04904fc8222933bc0e)

With this only being run once against a currently pretty limited dataset, I don't think it should matter much?

Reply:
Typically, you do these inside a rake task because you don't want to slow down a deploy by having it run this type of migration (which could fail). 

For our limited usage, this will work fine though.